CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# CONTRIBUTING
+
+## Setting up git hooks
+
+After you have cloned the repository for the first time, it is advised to set up git pre-commit hooks. These checks will also run on our CI infrastructure, but to save time, some lightweight checks can run locally before you commit ready to submit a PR.
+
+You only need to do this once!
+
+This repo uses [pre-commit](http://pre-commit.com/) to manage git pre-commit hooks. Installation of pre-commit is simplest via [homebrew](https://brew.sh/) or [linuxbrew](http://linuxbrew.sh/):
+
+```bash
+$ brew install pre-commit
+$ pre-commit install
```

Comment:
One final remark: I would add `--allow-missing-config` as option here, otherwise any commit without a `.pre-commit-config.yaml` will fail (e.g. in other git repos)